gips/data/merra/merra.py

```diff
@@ -24,9 +24,19 @@
 import os
 import datetime
 import time
-from pydap.client import open_url
 import numpy
 import signal
+import urllib2
+
+# this has to be done before open_url is imported
+from gips.data.merra.install_basic_client import install_basic_client
+from gips.settings import REPOS
+USER = REPOS['merra']['username']
+PASSWD = REPOS['merra']['password']
+install_basic_client('urs.earthdata.nasa.gov', USER, PASSWD, False)
```

Comment:
Is it necessary to have this code at module-scope?  In general that's frowned upon because it means this module can't be imported without triggering client installation.  There's only one call to open_url so you can put this call and `from pydap.client import open_url` right next to it down on line 247.  (And the REPOS access too.)